src/test/resources/broker.json

```diff
@@ -19,7 +19,8 @@
   "syslog": false,
   "stickiness": {
     "period": "10m",
-    "hostname": "host1"
+    "hostname": "host1",
+    "port": 1234
```

Comment:
Leave this file as it is, its to test backwards compat with older (pre 0.10.0) serialized JSON.